.github/workflows/build.yml

```diff
@@ -17,22 +17,22 @@ jobs:
     steps:
       - name: Checkout PR
         if: github.event_name == 'pull_request'
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4.1.1
```

Comment:
I would set @4 for rolling releases instead of pinning the version as we do it for all ther actions too.

Reply:
👍

Reply:
Did it get lost on rebasing? Still not "@4"

Reply:
now it is. gh actions do submit updated license info commits in this repo. change was not pushed.  now it should be there...